Minio/ApiEndpoints/BucketOperations.cs

```diff
@@ -441,5 +441,109 @@ public IObservable<MinioNotificationRaw> ListenBucketNotificationsAsync(string b
               });
 
         }
+
+        /// <summary>
+        /// Check if versioning is enabled on a bucket
+        /// </summary>
+        /// <param name="bucketName">Name of the new bucket</param>
+        /// <param name="cancellationToken">Optional cancellation token to cancel the operation</param>
+        /// <returns>An observable</returns>
+        public async Task<bool> IsVersioningEnabledAsync(string bucketName, CancellationToken cancellationToken = default(CancellationToken))
+         {
+            try
+            {
+                if (bucketName == null)
+                {
+                    throw new InvalidBucketNameException(bucketName, "bucketName cannot be null");
+                }
+                var request = await this.CreateRequest(Method.GET, bucketName).ConfigureAwait(false);
+                request.AddQueryParameter("versioning","");
+                var response = await this.ExecuteTaskAsync(this.NoErrorHandlers, request, cancellationToken).ConfigureAwait(false);
+            }
```

Comment:
you will need to read the versioningconfiguration xml and return the status here.

Reply:
Yes, I'd pushed the wrong copy. Thanks..